web/jobs_controller.go

```diff
@@ -35,18 +35,18 @@ func (jrc *JobsController) Index(c *gin.Context) {
 // Create adds the Jobs to the given context.
 // Example:
 //  "<application>/jobs"
-func (jc *JobsController) Create(c *gin.Context) {
+func (jrc *JobsController) Create(c *gin.Context) {
```

Comment:
`jc` is an abbreviation of `JobsController`.
`jrc` is an abbreviation of `JobRunsController`.

It looks like you should change `Index` to have a `jc *JobsController` receiver rather than this.